nerdlandbot/scheduler/PurgeScheduler.py

```diff
@@ -23,7 +23,9 @@ async def purge_messages(bot):
                 channel = bot.get_channel(int(text_channel))
                 try:
                     if channel:
-                        await channel.purge(check=check, before=before)
+                        deleted_messages = [1]
+                        while len(deleted_messages) > 0:
```

Comment:
I'd prefer to see a boolean variable, initialized to True. Then updated each iteration of the while loop to len(...) > 0